dashboard/app/helpers/users_helper.rb

```diff
@@ -265,7 +265,7 @@ def user_summary(user)
       sl.level_ids.each do |level_id|
         # if we have a contained level or BubbleChoice level, use that to represent progress
         level = Level.cache_find(level_id)
-        sublevel_id = sl.bubble_choice? ? level.best_result_sublevel(user)&.id : nil
+        sublevel_id = level.is_a?(BubbleChoice) ? level.best_result_sublevel(user)&.id : nil
```

Comment:
bug-fix-detour: I was checking if the script_level was BubbleChoice, but should have been checking the level itself